Tests/FluentAssertions.Specs/Primitives/BooleanAssertionSpecs.cs

```diff
@@ -109,5 +109,39 @@ public void Should_fail_with_descriptive_message_when_asserting_boolean_value_to
             action.Should().Throw<XunitException>()
                 .WithMessage("*Expected*boolean*True*because we want to test the failure message, but found False.*");
         }
+
+        [Fact]
+        public void Should_succeed_when_asserting_boolean_value_not_to_be_equal_to_the_same_value()
+        {
+            // Act
+            Action action = () =>
+                true.Should().NotBe(false);
+
+            // Assert
+            action.Should().NotThrow();
+        }
+
+        [Fact]
+        public void Should_fail_when_asserting_boolean_value_not_to_be_equal_to_a_different_value()
```

Comment:
You don't need this one since you already have a test with `because` and one without.

Reply:
The same could be said for the existing `Should_fail_when_asserting_boolean_value_to_be_equal_to_a_different_value` of which this is a copy?

Reply:
Fair point.